sim/paladin/retribution/presets.go

```diff
@@ -5,165 +5,178 @@ import (
+	CrusaderStrike:                true,
+	Precision:                     3,
+	DivineStrength:                5,
 }
 
-// Need to create a standard Retribution talents
-var StandardTalents = &proto.PaladinTalents{
-	Benediction:               5,
-	ImprovedSealOfTheCrusader: 3,
-	ImprovedJudgement:         2,
-	Conviction:                5,
-	SealOfCommand:             true,
-	Crusade:                   3,
-	SanctityAura:              true,
-	ImprovedSanctityAura:      2,
-	Vengeance:                 5,
-	SanctifiedJudgement:       2,
-	SanctifiedSeals:           0,
-	Fanaticism:                5,
-	CrusaderStrike:            true,
-	Precision:                 3,
-	DivineStrength:            5,
+var defaultRetRotation = &proto.RetributionPaladin_Rotation{
+	ConsecrationRank: proto.RetributionPaladin_Rotation_None,
+	UseExorcism:      false,
 }
 
-var PlayerOptionsBasic = &proto.Player_RetributionPaladin{
+var defaultRetOptions = &proto.RetributionPaladin_Options{
+	Judgement:             proto.RetributionPaladin_Options_Crusader,
+	CrusaderStrikeDelayMs: 1700,
+	HasteLeewayMs:         100,
+	DamageTakenPerSecond:  0,
+}
+
+var DefaultOptions = &proto.Player_RetributionPaladin{
 	RetributionPaladin: &proto.RetributionPaladin{
-		Talents:  StandardTalents,
-		Options:  retPalOptions,
-		Rotation: retPalRotation,
+		Talents:  defaultRetTalents,
+		Options:  defaultRetOptions,
+		Rotation: defaultRetRotation,
 	},
 }
 
-var retPalRotation = &proto.RetributionPaladin_Rotation{}
-
-var retPalOptions = &proto.RetributionPaladin_Options{}
-
 var FullRaidBuffs = &proto.RaidBuffs{
 	ArcaneBrilliance: true,
 	GiftOfTheWild:    proto.TristateEffect_TristateEffectImproved,
+	DivineSpirit:     proto.TristateEffect_TristateEffectImproved,
 }
 
 var FullPartyBuffs = &proto.PartyBuffs{
-	FerociousInspiration: 2,
+	Bloodlust:            1,
+	Drums:                proto.Drums_DrumsOfBattle,
+	BraidedEterniumChain: true,
+	ManaSpringTotem:      proto.TristateEffect_TristateEffectRegular,
+	StrengthOfEarthTotem: proto.StrengthOfEarthType_EnhancingTotems,
+	WindfuryTotemRank:    5,
 	BattleShout:          proto.TristateEffect_TristateEffectImproved,
-	LeaderOfThePack:      proto.TristateEffect_TristateEffectImproved,
-	SanctityAura:         proto.TristateEffect_TristateEffectImproved,
-	TrueshotAura:         true,
+	WindfuryTotemIwt:     2,
 }
 
 var FullIndividualBuffs = &proto.IndividualBuffs{
-	BlessingOfKings:  true,
-	BlessingOfWisdom: proto.TristateEffect_TristateEffectImproved,
-	BlessingOfMight:  proto.TristateEffect_TristateEffectImproved,
+	BlessingOfKings:     true,
+	BlessingOfMight:     proto.TristateEffect_TristateEffectImproved,
+	BlessingOfSalvation: true,
+	UnleashedRage:       true,
 }
 
 var FullConsumes = &proto.Consumes{
 	Flask:           proto.Flask_FlaskOfRelentlessAssault,
 	DefaultPotion:   proto.Potions_HastePotion,
-	SuperSapper:     true,
-	GoblinSapper:    true,
-	FillerExplosive: proto.Explosive_ExplosiveFelIronBomb,
```

Comment:
Include one of the explosives if you don't mind, just so that more things are tested by at least 1 sim.

Reply:
Updated, needed to remake ret tests accordingly